src/mockup-backend-evaluation/interfaces.ts

```diff
@@ -0,0 +1,45 @@
+import {ParameterInterface} from '../service/interfaces';
+
+// TODO: could or should be combined with PEAInterface in the future
+export interface PEAViewModelInterface {
+    peaId: string;
+    name: string;
+    connected: boolean;
+    status: Status;
+    alarmMessage: string;
+    type: string;
+    description: string;
+    parametersIn: ParameterInterface[];
+    parametersOut: ParameterInterface[];
+}
+
+export interface PEAHolderInterface {
+    id: string;
+    peaId: string;
+    type: string;
+    occupied: boolean;
+}
+
+export interface AlarmItemInterface {
+    position?: number;
+    timestamp: Date;
+    message: string;
+    id: string;
+    status: string;
+    name: string;
+}
+
+export interface PlantInfoInterface {
+    scenarioRunning: boolean;
+    receiptRunning: boolean;
+    plantCleaned: boolean;
+    currentScenario: string;
+}
+
+export enum Status {
```

Comment:
None = 'No Status available',
Normal = 'Normal',
PreWarning = 'PreWarning',
Warning = 'Warning',
Alert = 'Alert',
Malfunction = 'Malfunction'

Reply:
done, raised to alpha 15. This change will affect mockup-backend and frontend, which is not handled yet, so they still use alpha14.